projects/packages/backup-helper-script-manager/src/class-helper-script-manager.php

```diff
@@ -0,0 +1,598 @@
+<?php
+/**
+ * The Jetpack Backup Helper Script Manager class.
+ *
+ * @package automattic/jetpack-backup
+ */
+
+namespace Automattic\Jetpack\Backup;
+
+use Exception;
+
+/**
+ * Helper_Script_Manager manages installation, deletion and cleanup of Helper Scripts
+ * to assist with backing up Jetpack Sites.
+ *
+ * Does *not* use WP_Filesystem, because the helper script is something that we'll call over HTTP, so we don't want it
+ * to end up on an FTP server somewhere (even if it's the very same host). If there are permissions issues between the
```

Comment:
Thanks for the addition. I'd still suggest removing the old sentence that seems inaccurate though.
```suggestion
 * Does *not* use WP_Filesystem, because if there are permissions issues between the
```

Reply:
Fixed in e79e1c3.